src/html-imports.js

```diff
@@ -192,18 +192,16 @@
       // events can be done when all resources are ready.
       this.inflight = 0;
       this.dynamicImportsMO = new MutationObserver(m => this.handleMutations(m));
+      // Observe changes on <head>.
+      this.dynamicImportsMO.observe(document.head, {
```

Comment:
oh, it's just to delay the instantiation of Importer when the document is ready.